pkg/storage/gcp/gcp.go

```diff
@@ -32,38 +30,29 @@ type Storage struct {
 // See https://cloud.google.com/docs/authentication/getting-started.
 func New(ctx context.Context, gcpConf *config.GCPConfig, timeout time.Duration) (*Storage, error) {
 	const op errors.Op = "gcp.New"
-	storage, err := storage.NewClient(ctx)
+	s, err := storage.NewClient(ctx)
 	if err != nil {
 		return nil, errors.E(op, fmt.Errorf("could not create new storage client: %s", err))
 	}
-	bucketname := gcpConf.Bucket
-	u, err := url.Parse(fmt.Sprintf("https://storage.googleapis.com/%s", bucketname))
+
+	u, err := url.Parse(fmt.Sprintf("https://storage.googleapis.com/%s", gcpConf.Bucket))
 	if err != nil {
 		return nil, errors.E(op, err)
 	}
-	bkt := gcpBucket{storage.Bucket(bucketname)}
-	err = bkt.Create(ctx, gcpConf.ProjectID, nil)
-	if err != nil && !bucketExistsErr(err) {
+
+	bkt := s.Bucket(gcpConf.Bucket)
+	if _, err := bkt.Attrs(ctx); err != nil {
```

Comment:
Let's check if `err == storage.ErrBucketNotExist` and give a friendlier message in that case: return `errors.E(op, "You must manually create a storage bucket for Athens, see https://cloud.google.com/storage/docs/creating-buckets#storage-create-bucket-console")`

Reply:
Added but I don't know if I should break the line with log message (now is very long line)

Reply:
@marwan-at-work I think your comments have been addressed. Can we merge this?